src/3d/shaders/mesh/mesh.frag

```diff
@@ -16,12 +16,20 @@ uniform float lineWidth;
 uniform vec4 lineColor;
 
 uniform int textureType;
+
 uniform int colorRampType;
 uniform vec4 meshColor;
-
 uniform sampler1D colorRampTexture;
 uniform int colorRampCount;
 
+uniform bool arrowsEnabled;
+uniform sampler2D arrowsGridTexture;
+uniform vec2 arrowsMinCorner;
+uniform float arrowsSpacing;
+uniform sampler2D arrowTexture;
+uniform float maxVectorLength;
+uniform vec4 arrowsColor;
```

Comment:
can we have some comments for these new uniforms? For some it is particularly unclear what are the units of the variables... (and which ones are user-defined)